components/MhiAcCtrl/mhi_ac_ctrl.h

```diff
@@ -39,7 +39,9 @@ static const std::vector<std::string> protection_states = {
 
 static const char* TAG = "mhi_ac_ctrl";
 
-unsigned long room_temp_api_timeout_ms = millis();
+unsigned long room_temp_api_timeout_ms = esphome::millis();
```

Comment:
I suppose you could move this line within the esphome namespace block, so it's defined within the proper scope. Also, no need for the `esphome::` prefix if you do that.